pom.xml

```diff
@@ -51,7 +51,7 @@
     <projectEmail>nos-devel@redhat.com</projectEmail>
     <!--<undertowVersion>1.3.33.Final</undertowVersion>-->
     <atlasVersion>1.0.1</atlasVersion>
-    <partylineVersion>2.0</partylineVersion>
+    <partylineVersion>1.9.4</partylineVersion>
```

Comment:
Seems recent 1.x release version is 1.16 for partyline. A concern here is I'm not quite sure if there are some critical issues fix between 1.16 and 1.9.4.